### ISSUE TITLE
raku_unicode_abbrevs breaks //

### ISSUE THREAD

**Issue by zerodogg** [Raku/vim-raku#20]
When `raku_unicode_abbrevs` is enabled it becomes impossible to type `//`, when I type the second / it replaces the first one with ÷, which isn't quite what I want when I wanted to use defined-or.

**Comment by zostay** [comment]
Yeah, I recommend not using that feature. It's slated to be removed rather than fixed.
